src/app/shared/plnkr/plunker-data.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Plunker data
+ *   with information about Component name, selector, files used in example, and path to examples
+ */
+export class PlunkerData {
```

Comment:
Changed the name to ExampleData. I think we can extract information (selector, componentName) from example file. I will work on that in another PR.